### PR TITLE
Add EVENT_BRIDGE_RESOURCE_PREFIX into localconfig-example

### PR DESCRIPTION
**Pull request by jomarko**
NO_JIRA
## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] The layers in the `kustomize` folder have been updated accordingly.
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

**Review by kostola — APPROVED** [review]

File: dev/bin/localconfig-example [hunk 1]
```diff
@@ -10,6 +10,10 @@ MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN=my-offline-token
 
 # RHOAS_USERNAME=
 
+# This will configure the local RHOSE instance to use your username as prefix for Kafka topics and Managed Connectors
+# making it way easier to identify who is using it.
+EVENT_BRIDGE_RESOURCE_PREFIX=my_username
```
Comment (sutaakar) [inline dev/bin/localconfig-example:15]:
```suggestion
export EVENT_BRIDGE_RESOURCE_PREFIX=my_username
```

Comment (sutaakar) [reply dev/bin/localconfig-example:15]: AFAIK export is needed as the `EVENT_BRIDGE_RESOURCE_PREFIX` is not a script property but env variable applied directly as manager config.

Comment (kostola) [reply dev/bin/localconfig-example:15]: yes that's true...good catch @sutaakar :+1:

Comment (sutaakar) [reply dev/bin/localconfig-example]: hmm, when thinking about it again, maybe it is not needed.....
not sure whether the variables specified in localconfig are applied as env variables
I just followed the email instructions, which were mentioning export
may be worth trying without export

**Review by jomarko — COMMENTED** [review]
I think this version is correct, after last message on *[rhose-dev] Suggestion: use EVENT_BRIDGE_RESOURCE_PREFIX for local testing* email thread. 

@sutaakar @lucamolteni ?

**Review by sutaakar — APPROVED** [review]
Based on the email messages - approving

**Pull request merged**